src/index/vchordrq/scanners/maxsim.rs

```diff
@@ -72,6 +73,7 @@ impl SearchBuilder for MaxsimBuilder {
         options: SearchOptions,
         mut fetcher: impl Fetcher + 'b,
         bump: &'b impl Bump,
+        _sender: impl CollectorSender,
```

Comment:
I think you could try to support this here, respecting PostgreSQL's text representation of `vector[]`.